h/static/scripts/test/root-thread-test.js

```diff
@@ -276,22 +273,6 @@ describe('rootThread', function () {
       // pages, since we show all types of annotations here
       assert.notOk(threadFilterFn);
     });
-
-    it('filter does not match annotation when it is still waiting to anchor', function () {
```

Comment:
This test has been removed because it is now covered more directly in `tabs-test.js`